client/components/app-layout.vue

```diff
@@ -56,7 +56,7 @@
         fluid
         fill-height
       >
-        <router-view />
+        <router-view :key="$route.fullPath" />
```

Comment:
Why is this necessary?

Reply:
Suppose that an user is editing a document (*d1*) at the */document/d1* url and clicks on the Fork button. A new document (*d2*) is created and the user has to be redirected to */document/d2* (basically to the same page but displaying the new document). If `:key="$route.fullPath"` is not used in the `router-view` the document component won't be reloaded because we are routing to the same component.